integreat_cms/cms/views/users/user_form_view.py

```diff
@@ -100,6 +104,11 @@ def post(self, request, *args, **kwargs):
                     "A user has to be either staff/superuser or needs to be restricted to at least one region."
                 ),
             )
+        elif not current_user.is_superuser:
```

Comment:
```suggestion
        elif not request.user.is_superuser and "is_superuser" in user_form.changed_data:
```
The problem is that with your suggestion, non-superuser cannot save the form at all, which solves the privilege escalation but unfortunately we need normal staff users to be able to change other user fields.
Thus, I suggest to only raise the error when the non-superuser tries to change the `is_superuser`-field.

Reply:
Yeah, of course you're right...